libopae/plugins/xfpga/sysfs.c

```diff
@@ -487,6 +487,7 @@ int sysfs_finalize(void)
 		sysfs_region_destroy(&_regions[i]);
 	}
 	_sysfs_region_count = 0;
+	_sysfs_format_ptr = NULL;
```

Comment:
Are these protected by any kind of lock?

Reply:
oh, no.. 😞

Reply:
Could you address in another pull? :)